### ISSUE TITLE
Validation Loss graph

### ISSUE THREAD

**Issue by Annieliaquat** [tensorflow/models#10843]
I am using SSD MobileNet V2 FPNLite 320x320 for training. I have got Total_loss graph for training step of 4000. Drawing graph for training loss is easy as it gives total loss for every iteration, e.g (from 100 steps to 4000 steps)
 But I want to draw graph for validation loss which is 0.26. In validation loss it just gives the total loss. How should I draw my line graph for validation loss as I have just one point. There is a paper that have drawn graph for their validation loss but I don't know how.  Please I want the same curly graph for my model. 

![image](https://user-images.githubusercontent.com/43025113/204077617-7416b3c3-1f07-4414-94b6-85f67acff4ca.png)

**Comment by Annieliaquat** [comment]
> I am using SSD MobileNet V2 FPNLite 320x320 for training. I have got Total_loss graph for training step of 4000. Drawing graph for training loss is easy as it gives total loss for every iteration, e.g (from 100 steps to 4000 steps) But I want to draw graph for validation loss which is 0.26. In validation loss it just gives the total loss. How should I draw my line graph for validation loss as I have just one point. There is a paper that have drawn graph for their validation loss but I don't know how. Please I want the same curly graph for my model.
> 
> ![image](https://user-images.githubusercontent.com/43025113/204077617-7416b3c3-1f07-4414-94b6-85f67acff4ca.png)
